### ISSUE TITLE
Object.OktaAuthModule_Factory - Error: inject() must be called from an injection context

### ISSUE THREAD

**Issue by Angelotas** [okta/okta-angular#88]
### Describe the bug?

I have an Angular Elements based application (building web components through Angular) where I implemented okta authentication using the angular `ng add @oktadev/schematics` scheme. If I run the component standalone no problem, but when I try to import it into another Angular app it throws the following error.
![image](https://user-images.githubusercontent.com/18513790/156356577-639a6f8b-f8f2-4b54-8ffc-fd6ac164f5f9.png)

Error is shown importing `OktaAuthModule` in `app.module.ts`.

I have been looking for the error and the most I have seen are some fixes like:
- In your application, add to the angular.json file the property preserveSymlinks: true in `projects.$name.architect.build.options`.
- In your tsconfig.json add
`"routes": { "@angular/*": [ "./node_modules/@angular/*" ] }`

It did not work for me.

### What is expected to happen?

I didn't expect issues like standalone execution mode.

### What is the actual behavior?

```
Error: Uncaught (in promise): Error: NG0203: inject() must be called from an injection context
Error: NG0203: inject() must be called from an injection context
    at injectInjectorOnly (core.mjs:4756:15)
    at Module.ɵɵinject (core.mjs:4766:12)
    at Object.OktaAuthModule_Factory [as factory] (okta-angular.js:317:108)
    at R3Injector.hydrate (core.mjs:11435:1)
    at R3Injector.get (core.mjs:11251:1)
```

### Reproduction Steps?

Create a simple angular element that includes Okta using ng add @oktadev/schematics (you have to provide an issuer and clientId). Then you try to import it into another Angular application and the error should appear.

### SDK Versions

Angular CLI: 13.1.4
Node: 16.13.0
Package Manager: npm 8.1.0
OS: win32 x64

Angular: 13.1.3
... animations, common, compiler, compiler-cli, core, elements
... forms, platform-browser, platform-browser-dynamic, router

```
    "@angular/core": "~13.1.0",
    "@angular/elements": "^13.1.3",
.....
    "@okta/okta-angular": "5.1.1",
    "@okta/okta-auth-js": "6.0.0",
```

### Execution Environment

Browser:
- [X] Chrome (desktop) version 68.0.3440.106
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [X] Firefox version 61.0.2
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [X] Edge version 42.17134.1.0

**Comment by jaredperreault-okta** [comment]
Thank you for the report! We will look into this.

Internal Ref: OKTA-475810

**Comment by Angelotas** [comment]
Thanks @jaredperreault-okta.

For the record, I kept looking for a solution and found that when I import Okta into an Angular 12 element instead of Angular 13 element, there is no problem when I inject it into an Angular 13 app.

**Comment by Angelotas** [comment]
Hi @jaredperreault-okta.

I found the problem! I was related with Module Federation Plugin. When I load the Angular Element component into another Angular app, I'm using module federation and since both are using same version of Angular (specifically 13), I had to indicate that my Angular Element needs and singleton instance of Angular:

**webpack.config.js**
```
...
plugins: [new ModuleFederationPlugin({
...
  shared: {
            "@angular/core": { eager: true, singleton: true },
            "@angular/common": { eager: true, singleton: true },
            "@angular/router":  { eager: true, singleton: true },
  },
...
})
   
```

So I avoid `Error: NG0203: inject() must be called from an injection context` caused by there are several Angular versions loaded at once
